DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Geometry/ISpeckleObjectToHost/LineToHostConverter.cs

```diff
@@ -0,0 +1,25 @@
+using Speckle.Converters.Common;
+using Speckle.Converters.Common.Objects;
+using Speckle.Core.Models;
+
+namespace Speckle.Converters.ArcGIS3.Geometry.ISpeckleObjectToHost;
+
+[NameAndRankValue(nameof(SOG.Line), NameAndRankValueAttribute.SPECKLE_DEFAULT_RANK)]
+public class LineSingleToHostConverter : ISpeckleObjectToHostConversion, IRawConversion<SOG.Line, ACG.Polyline>
+{
+  private readonly IRawConversion<SOG.Point, ACG.MapPoint> _pointConverter;
+
+  public LineSingleToHostConverter(IRawConversion<SOG.Point, ACG.MapPoint> pointConverter)
+  {
+    _pointConverter = pointConverter;
+  }
+
+  public object Convert(Base target) => RawConvert((SOG.Line)target);
+
+  public ACG.Polyline RawConvert(SOG.Line target)
+  {
+    List<SOG.Point> originalPoints = new() { target.start, target.end };
+    IEnumerable<ACG.MapPoint> points = originalPoints.Select(x => _pointConverter.RawConvert(x));
+    return new ACG.PolylineBuilderEx(points, ACG.AttributeFlags.HasZ).ToGeometry();
```

Comment:
Is there no simple line in ArcGIS?

Reply:
nope: https://pro.arcgis.com/en/pro-app/latest/sdk/api-reference/topic8158.html